feature/platform/transceiver/zr_tunable_parameters_test/README.md

```diff
@@ -0,0 +1,137 @@
+      * Both transceivers states are enabled.
+      * Validate setting 400ZR optics module tunable laser center frequency
+        across frequency range 196.100 - 191.400 THz for 100GHz grid.
+      * Validate setting 400ZR optics module tunable laser center frequency
+        across frequency range 196.100 - 191.375 THz for 75GHz grid.  
+      * Specific frequency details can be found in 400ZR implementation
+        agreement under sections 15.1 ad 15.2 Operating frequency channel
+        definitions.
+      * Validate adjustable range of transmit output power across -13 to -9 dBm
+        range in steps of 1dB. So the module’s output power will be set to -13,
+        -12, -11, -10, -9 dBm in each step.
+
+*   With the ZR link established as explained above, for each configured
+    frequency and TX output power value verify that the following ZR
+    transceiver telemetry paths exist and are streamed for both the ZR optics.
+    * Frequency
+        *   /components/component/optical-channel/state/frequency
+        *   /components/component/optical-channel/state/carrier-frequency-offset/instant
+        *   /components/component/optical-channel/state/carrier-frequency-offset/avg
+        *   /components/component/optical-channel/state/carrier-frequency-offset/min
+        *   /components/component/optical-channel/state/carrier-frequency-offset/max
+    * TX Output Power
+        *   /components/component/optical-channel/state/output-power/instant
+        *   /components/component/optical-channel/state/output-power/avg
+        *   /components/component/optical-channel/state/output-power/min
+        *   /components/component/optical-channel/state/output-power/max
+
+* With above streamed data verify
+    * For each center frequency, laser frequency offset should not be more than
+      +/- 1.8 GHz max.
+    * When set to a specific target output power, transmit power control
+      absolute accuracy should be within +/- 1 dBm of the target value.
+    * For reported data check for validity: min <= avg/instant <= max
+
+   
+## TRANSCEIVER-5.2
+
+*   When the modules or the devices are still in a boot stage, they must not
+    stream any invalid string values like "nil" or "-inf".
+
+*   Frequency must be specified as uint64 in MHz. Streamed values for frequency
+    offset must be of type decimal64.
+
+*   TX Output power must be of type decimal64.
+
+## TRANSCEIVER-5.3
+
+*   Verify that the optics Tunable Frequency and TX output power tunes back to
+    the correct value as per configuration after the interface flaps.
+
+    *   Enable a pair of ZR interfaces on the DUT as explained above.
+    *   Verify the ZR optics frequency and TX output power telemetry values are
+        set in the normal range.
+    *   Disable or shut down the interface on the DUT.
+    *   Verify with interfaces in down state both optics are streaming uint 0
+        value for frequency and decimal64 for TX output power.
```

Comment:
Is 0 the expected output power?  Since output power is measured in dBm, doesn't 0dBm = 1 milliwatt?  Seems like there should be no power for a disabled interface.    Maybe the output power for a down interface should be  -99?  I've seen that implemented in the past.

Reply:
good catch, zero should be for frequency only. Updated to call out as you suggested